src/app/address/steps.js

```diff
@@ -20,7 +21,20 @@ module.exports = {
         value: true,
         next: "results",
       },
-      "address",
+      "problem",
+    ],
+  },
+  "/problem": {
```

Comment:
Not sure on the name of this page.

Reply:
`not-found`? Or is that too confusing with 404?

Maybe this is good enough for now, and you can have 5 minute workshop with Tom when he's back on naming things?